client/src/app/app.module.ts

```diff
@@ -17,9 +17,9 @@ import { CookieService } from 'ngx-cookie-service';
     BrowserModule,
     BrowserAnimationsModule,
     AuthModule,
-    AuthRoutingModule,
     AltaMainModule,
     AltaMainRoutingModule,
+    AuthRoutingModule,
     HttpClientModule,
```

Comment:
I think you forgot to add here the module related to angular material that you just created